### PR TITLE
Create Docs

### PR DESCRIPTION
**Pull request by Mineinjava**
I created the docs. These docs *should* be able to run on ReadTheDocs.

TODO:

- [ ] add more docstrings

---
Made with `sphinx.ext.autodoc`.

All docstrings must use ReStructuredText format.

### CODE REVIEW

**Review by LennyPhoenix — APPROVED** [review]
Wow! Nice work on getting all that set up, should I merge this in now or do you want to add docstrings first?

File: docs/discordsdk.rst [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+discordsdk.enum module
+----------------------
+
+.. automodule:: discordsdk.enum
+   :members:
+   :undoc-members:
+   :show-inheritance:
+
+discordsdk.event module
+-----------------------
```
Comment (LennyPhoenix) [inline docs/discordsdk.rst:48]: This module is only used internally, so we might not need to put this on the auto-docs.

Comment (Mineinjava) [reply docs/discordsdk.rst]: done!

**Comment by Mineinjava** [comment]
Lets merge this before adding docstrings.

**Pull request merged**